proposals/2022-07-currency-allow-list/README.md

```diff
@@ -0,0 +1,31 @@
+# Currency Allow List Criteria
+
+This is a text proposal for the community to adopt a set of criteria as guidelines when evaluating future governance proposals to enable specific currencies in Regen Network's on-chain marketplace.
+
+- A Yes result on the proposal would provide a clear signal that the Regen Network community accepts the following as guidelines for evaluation criteria when discussing and voting to enable currencies in the allow list for use in Regen Network's ecocredit marketplace.
+
+- A No result on the proposal would force reconsideration of the criteria as outlined here.
+
+## Background
+
+Regen Ledger v4.0 includes a new on-chain marketplace that enables ecocredits to be listed for sale by their owners. In other words, a user who has ecocredits (unretired) in their account can list any number of their credits for sale (creating a standing "sell order"). The credits are escrowed on-chain, and another user can purchase up to the full quantity of credits in the sell order at the specified price.
+
+In order to prevent fragmented markets and an over-complicated end-user experience, credits cannot be listed in the marketplace in any currency at all. Rather, the listed of allowed "currencies" that ecocredits can be listed for sale in is controlled through on-chain governance.
```

Comment:
```suggestion
In order to prevent fragmented markets and an over-complicated end-user experience, credits cannot be listed in the marketplace in any currency at all. Rather, the listed of allowed "currencies" that ecocredits can be listed for sale is controlled through on-chain governance.
```